tests/Install-Scripts.Test/AkaMsLinksTests.cs

```diff
@@ -277,6 +277,85 @@ public void LinkCanBeCreatedForGivenRuntime(string channel, string runtime, stri
             commandResult.Should().HaveStdOutContaining(output => Regex.IsMatch(output, expectedLinkPattern));
         }
 
+        [Theory]
+        [InlineData("3.1", null, @"https://aka.ms/dotnet/internal/3.1/dotnet-sdk-")]
+        [InlineData("5.0.2xx", null, @"https://aka.ms/dotnet/internal/5.0.2xx/dotnet-sdk-")]
+        [InlineData("Current", null, @"https://aka.ms/dotnet/internal/current/dotnet-sdk-")]
+        [InlineData("LTS", null, @"https://aka.ms/dotnet/internal/LTS/dotnet-sdk-")]
+        [InlineData("5.0.2xx", "validated", @"https://aka.ms/dotnet/internal/5.0.2xx/validated/dotnet-sdk-")]
+        public void InternalLinkCanBeCreatedForSdk(string channel, string quality, string expectedLink)
```

Comment:
I think 'internal' can be just a parameter to existing test methods.

Reply:
Well, I can unite these tests.